packages/patrol_cli/lib/src/runner/patrol_command.dart

```diff
@@ -181,6 +181,20 @@ abstract class PatrolCommand extends Command<int> {
     );
   }
 
+  void usesShowFlutterLogs() {
+    argParser.addFlag(
+      'show-flutter-logs',
+      help: 'Show Flutter logs while running the tests.',
+    );
+  }
+
+  void usesShowTestSteps() {
+    argParser.addFlag(
+      'show-test-steps',
+      help: 'Show test steps while running the tests.',
+    );
```

Comment:
I think we should consider setting this one to `true` by default. Default behaviour on Android doesn't give any information about passed/failed test.

Reply:
This flag steer only showing test steps, the test status is always displayed. But as we discussed internally, I will change this flag to hide-test-steps